internal/api/web/handlers/quiz.go

```diff
@@ -12,13 +15,42 @@ import (
 func RegisterQuizHandlers(e *echo.Echo) {
 	e.GET("/", quizHomePage)
 	e.GET("/quizpage", GetQuizPage)
+	e.GET("/nextquestion", GetIsCorrect)
+	e.POST("/nextquestion", POSTNextQuestion)
 }
 
 // Renders the quiz home page
 func quizHomePage(c echo.Context) error {
 	return utils.Render(c, http.StatusOK, quiz_pages.QuizHomePage())
 }
 
+var questionIndex = 0
```

Comment:
Since we are using REST style API, State should not be handled in the the API call.